src/shared/Microsoft.Git.CredentialManager/Constants.cs

```diff
@@ -13,6 +13,39 @@ public static class Constants
         public const string ProviderIdAuto  = "auto";
         public const string AuthorityIdAuto = "auto";
 
+        public static class RegexPatterns
```

Comment:
I haven't searched extensively but I only found usage of `Any`, `Empty`, and `None`. Shall we remove the unused patterns?

Reply:
Sure.